docs/pages/project/contributing/contributing-ui-tests.md

```diff
@@ -38,33 +41,38 @@ There are several tools to help you to working with environment variables locall
 
 There are a few ways to set up the Meshery UI and server, but for end-to-end testing, we aim to get as close to a production environment as possible. We know developers might need to make some tweaks for UI and Server. Rebuilding the whole project can take time, and we don’t support hot reload because it’s more for development than for end-to-end testing.
 
-{% include alert.html type="warning" content="Some test cases required you to have kubernetes cluster and build meshery adapter as well, be aware of that. Which is out of scope for this documentation" %}
+{% include alert.html type="warning" title="Several Test may break" content='Some test cases required you to have kubernetes cluster and build meshery adapter as well, be aware of that. Which is out of scope for this documentation<ul><li><a href="https://docs.meshery.io/installation/kubernetes/minikube">Kubernetes Cluster</a>: Instalation of kubernetes cluster with Minikube.</li>
+<li><a href="https://docs.meshery.io/installation/multiple-adapters">Meshery Adapters</a>: Using Multiple Adapters</li></ul>' %}
 
-### Native OS
+### Native OS (Recommended)
 
 This approach is very quick to build, but also dependent on your operating system, so you need to have all dependencies necessary to be able compile and running the server.
 
-- Install & Build the NextJS static site generator application for both the UI and UI Provider
+- Install & Build the NextJS application for both the UI and UI Provider
 
 ```bash
 make ui-build
 ```
 
-- Compile golang server
+- Compile the Golang into binary file for Meshery Server
 
 ```bash
 make build-server
 ```
 
-- Run the server locally on port 8080
+- Run the Meshery Server on localhost port 9081
 
 ```bash
 make server-binary
 ```
 
+### Meshery CLI
+
+There is also Meshery CLI which can help you run the UI and Server, for more detail, you go to [Meshery CLI documentation](https://docs.meshery.io/project/contributing/contributing-cli-guide#process)
+
 ### Docker Based
 
-Alternatively, a Docker-based setup can be utilized, which simplifies the process, ensures consistency across different environments and also more close to production environment as well compare to native solution. 
+Alternatively, a Docker-based setup can be utilized, simplifying the process, and ensuring consistency across different environments. It is closer to the production environment than the native solution but slower in terms of build time.
```

Comment:
Hey Jerens, thanks for the changes. Curious as to why native os is flagged as recommended however this description notes this is closer to prod env?